_posts/en/newsletters/2019-04-16-newsletter.md

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Bitcoin Optech Newsletter #42'
+permalink: /en/newsletters/2019/04/16/
+name: 2019-04-16-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter requests testing of the latest release candidates
+for Bitcoin Core and LND, describes how helping people accept payments
+to bech32 addresses can lower fees, and lists notable code changes in
+popular Bitcoin projects.
+
+## Action items
+
+- **Help test Bitcoin Core 0.18.0 release candidates:** Bitcoin Core's
+  third RC for its next major version is [available][0.18.0] and a
+  fourth is being prepared.  Testing is greatly appreciated.  Please use
+  [this issue][Bitcoin Core #15555] for reporting feedback.
+
+- **Help test LND 0.6-beta RC4:** [release candidates][lnd releases]
+  for the next major version of LND are being published.  Testing by
+  organizations and experienced LN users is encouraged to catch any
+  regressions or serious problems that could affect users of the final
+  release.  [Open a new issue][lnd issue] if you discover any problems.
+
+## News
+
+*No notable technical news this week.  (Note: When we at Optech started
+this newsletter, we decided to avoid stuffing short newsletters with
+fluff pieces and other unnecessary information, so newsletter length
+varies depending on the actual amount of significant technical news each
+week.  You've probably seen us publish an occasional very long
+newsletter; this week, you see the opposite.)*
+
+## Bech32 sending support
+
+*Week 5 of 24.  Until the second anniversary of the segwit soft
+fork lock-in on 24 August 2019, the Optech Newsletter will contain this
+weekly section that provides information to help developers and
+organizations implement bech32 sending support---the ability to pay
+native segwit addresses.  This [doesn't require implementing
+segwit][bech32 easy] yourself, but it does allow the people you pay to
+access all of segwit's multiple benefits.*
+
+One reason your users and customers may want you to implement bech32
+sending support is because it'll allow the receivers of those payments
+to save on fees when they re-spend that money.  This week, we'll look at
```

Comment:
Totally optional but could perhaps mention that there are use cases where you spend to yourself (ie UTXO consolidation) in which you, the sender, are the recipient of this benefit.

Reply:
This is a good idea, but I looked at it and think it weighs down the introduction too much.